### ISSUE TITLE
aria-colindex and aria-colspan combination doesn't work

### ISSUE THREAD

**Issue by gportela85** [FreedomScientific/VFO-standards-support#401]
If I have a table with nested headers, the parent headers are not recognized in order.

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Go to https://jsfiddle.net/8pjg1fao/
> 2. Click on `Cell Text 2` under `Field 2`

### Expected result

> Expected JAWS to read `Group 1` ->  `Field 2` -> `Cell Text 2`

### Actual result

> JAWS read `Group 2` -> `Field 2` -> `Cell Text 2`

#### JAWS version and build number

JAWS 2020.2006.12
 
#### Operating System and version
Windows 10

#### Browser and version:

Chrome 83

**Comment by JeremiahRogersVispero** [comment]
Bug 192036
